runtime/serde/serde-benchmarks/models/twitter.smithy

```diff
@@ -0,0 +1,205 @@
+$version: "1.0"
+
+namespace aws.benchmarks.twitter
+
+use aws.api#service
+use aws.protocols#restJson1
+
+@service(sdkId: "Twitter JSON Benchmark")
```

Comment:
comment

I wonder if any of this work is upstreamable to Smithy.  It would seem that any client SDK generator would want to measure performance of at least serde.  OTOH things outside of the scope of codegen would also want to be benchmarked so not a perfect fit for a Smithy-based solution.  :shrug: